translation/en.json

```diff
@@ -1,7 +1,7 @@
 {
-	"DISCLAIMER": "This article is issued from ${creator}. The text is licensed under ${license}. Additional terms may apply for the media files.",
+	"DISCLAIMER": "This article is issued from ${creator}. The text is available under ${license} unless otherwise noted. Additional terms may apply for the media files.",
 	"LAST_EDITED_ON": "Last edited on ${date}",
-	"LICENSE_NAME": "Creative Commons - Attribution - Sharealike",
+	"LICENSE_UNKNOWN": "an unknown license",
```

Comment:
You must:
- remove all `LICENSE_NAME` from all other locale JSON files (this is not needed anymore)
- add `LICENSE_UNKNOWN` to `qqq.json` with an explanation about the meaning of this key (`qqq.json` is a "magic" file used by TranslateWiki to load help text for translators